src/Stratis.Bitcoin/P2P/PeerAddress.cs

```diff
@@ -12,6 +12,17 @@ namespace Stratis.Bitcoin.P2P
     [JsonObject]
     public sealed class PeerAddress
     {
+        /// <summary>
+        /// The maximum amount of times a peer can be attempted within a give time frame.
+        /// </summary>
+        internal static readonly int AttemptThreshold = 10;
```

Comment:
Perhaps 10 attempts is too much per period?

Reply:
OK I agree! Lets make it 5?